README.md

```diff
@@ -1,2 +1,222 @@
 # stellar-dbt-public
 Public DBT instance to aid in data transformation for analytics purposes
+
```

Comment:
Can you add a sentence that links to the dbt docs for those interested in setting up their own project?